.github/build-and-test

```diff
@@ -39,7 +39,7 @@ declare -ra dirs=(
 for dir in "${dirs[@]}"
 do
     pushd $dir
-    if ! bazel test --config=${BAZEL_CONFIG_FLAG:-''} //... ; then
+    if ! bazel test --config=${BAZEL_CONFIG_FLAG:-ci} //... ; then
```

Comment:
I think a folder `tools/` would make the most sense, as I see that pattern quite often. Perhaps also have the script be a bit more parallel? So that the different build and tests happen together by spawning multiple processes?

Maybe a broader question, should we consider moving the `build-and-test` script out of the GitHub actions completely, and just have it be a local development nicety? Then we can refactor the GitHub actions a bit and use more of it features.

Reply:
> I think a folder tools/ would make the most sense, as I see that pattern quite often

Yeah, `tools` sound good to me too.

> Perhaps also have the script be a bit more parallel? So that the different build and tests happen together by spawning multiple processes?

Bazel already leverages system resources quite well, it would be interesting to see what the effect of running things in parallel is. But I'd do that in a separate PR.

> Maybe a broader question, should we consider moving the build-and-test script out of the GitHub actions completely, and just have it be a local development nicety? Then we can refactor the GitHub actions a bit and use more of it features.

What do you have in mind? Maybe create an issue were we can discuss this...